### DISCUSSION TITLE
Help to import HTML into Grafana

### DISCUSSION THREAD

**Discussion by TFabio69** [grafana/grafana-infinity-datasource#184] [Q&A]
Hi, i have this file html:
"TagId";"TagName";"Value";"AlStatus";"AlType";"Quality" 2;"SpeedSetPoint";0;0;0;65472 4;"SetTemperatureWarmCylinder";100;0;0;65472 6;"SetPressureDancer";85;0;0;65472 7;"SetEntryWiderTension";85;0;0;65472 9;"SetExitBeltTension";10;0;0;65472 11;"SetpointDephase";15;0;0;65472 12;"ActSpeedDrive1";0;0;0;65472 19;"ActSpeedDrive8";0;0;0;65472 20;"ActTorqueDrive1";0;0;0;65472 21;"ActTorqueDrive2";0;0;0;65472 22;"ActTorqueDrive3";0;0;0;65472 23;"ActTorqueDrive4";0;0;0;65472 24;"ActTorqueDrive5";0;0;0;65472 25;"ActTorqueDrive6";0;0;0;65472 26;"ActTorqueDrive7";0;0;0;65472 27;"ActTorqueDrive8";0;0;0;65472 28;"ActSpeedMotor1";0;0;0;65472 29;"ActSpeedMotor2";0;0;0;65472 30;"ActSpeedMotor3";0;0;0;65472 31;"ActSpeedMotor4";0;0;0;65472 32;"ActSpeedMotor5";0;0;0;65472 33;"ActSpeedMotor6";0;0;0;65472 34;"ActSpeedMotor7";0;0;0;65472 35;"ActSpeedMotor8";0;0;0;65472 36;"EMU_Current";2.25;0;0;65472 37;"EMU_Voltage";394.4;0;0;65472 59;"ActSpeedDrive2";0;0;0;65472 60;"ActSpeedDrive3";0;0;0;65472 61;"ActSpeedDrive4";0;0;0;65472 62;"ActSpeedDrive5";0;0;0;65472 63;"ActSpeedDrive6";0;0;0;65472 64;"ActSpeedDrive7";0;0;0;65472 66;"ActTemperatureWarmCylinder";36.9;0;0;65472 67;"Countermeter_total";137266.2;0;0;65472 68;"Countermeter_daily";282769.5;0;0;65472 69;"ActExitBeltTension";11.64892;0;0;65472 70;"ActEntryWiderTension";75.7561;0;0;65472 71;"ActLineSpeed";0;0;0;65472 72;"ActPosBeltLeftPress";-0.4934523;0;0;65472 73;"ActPosBeltRightPress";-0.1699877;0;0;65472
where each line is a istantaneous tag value and I wanto to show in the grafana dashboard.
I receive null value when try to import this data.
Someone can help me to set correctly the data source ?

Thanks for attention
Fabio

**Comment by yesoreyeram** [comment]
This doesn't look properly formatted. Is this a CSV file with `;` delimited? If so, you need to use CSV option and `;` as your delimiter in CSV option.

**Comment by TFabio69** [comment]
Is an html file copied from a browser window.


Fabio

Da: Sriram ***@***.***>
Inviato: mercoledì 27 ottobre 2021 13:43
A: yesoreyeram/grafana-infinity-datasource ***@***.***>
Cc: Fabio Tiozzo ***@***.***>; Author ***@***.***>
Oggetto: Re: [yesoreyeram/grafana-infinity-datasource] Help to import HTML into Grafana (Discussion #184)


This doesn't look properly formatted. Is this a CSV file with ; delimited? If so, you need to use CSV option and ; as your delimiter in CSV option.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyesoreyeram%2Fgrafana-infinity-datasource%2Fdiscussions%2F184%23discussioncomment-1544788&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7Cba90ae3d2e8a476a6c3c08d9993ee312%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709317759622783%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=w9ovTe2PSbyAde7pfYXVlwoANJz9R2M%2FCChS9eOEQVA%3D&reserved=0>, or unsubscribe<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWHPPVJOXDUJYVV3J6ZTHCTUI7XT3ANCNFSM5GZ5F5VA&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7Cba90ae3d2e8a476a6c3c08d9993ee312%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709317759632778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=zX9XfFboFbpCeJrDTqsYRXVqtJ%2FwFI7AVzte9FVWS7I%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7Cba90ae3d2e8a476a6c3c08d9993ee312%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709317759632778%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ankKQY4%2BGdH65dWX4xsyYFgGmwpHKEM1wVNVTnd3vsg%3D&reserved=0> or Android<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7Cba90ae3d2e8a476a6c3c08d9993ee312%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709317759642777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=%2FzRNQKDiRYXZPKyvJTu5uU%2BIX8dqHPmiNzG1mIkZmbQ%3D&reserved=0>.

________________________________
CONFIDENTIALITY NOTICE
This  message  and   its   attachments   are  addressed  solely  to  the  persons above  and   may  contain  confidential  information.  If  you  have  received  the message  in  error, be informed that any use of the content hereof is prohibited. Please return it immediately to the sender and delete the message. Should you have any questions, please contact us by replying to Santex Rimar ***@***.***>
Thank you

NOTIZIA CONFIDENZIALE
Questo  messaggio  e  i  suoi allegati  sono  indirizzati alle persone scritte sopra. Se  avete  ricevuto  questo  messaggio per errore, Vi informiamo che l'uso delle informazioni  contenute   è   proibito.  Prego  rinviate al  mittente immediatamente. Per altri problemi inviare una e-mail a Santex Rimar ***@***.***>
Grazie

**Reply by yesoreyeram** [reply]
Something like [this](https://grafana-infinity-datasource.herokuapp.com/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Infinity%22,%7B%22refId%22:%22A%22,%22type%22:%22csv%22,%22source%22:%22inline%22,%22format%22:%22table%22,%22data%22:%22%5C%22TagId%5C%22;%5C%22TagName%5C%22;%5C%22Value%5C%22;%5C%22AlStatus%5C%22;%5C%22AlType%5C%22;%5C%22Quality%5C%22%5Cn2;%5C%22SpeedSetPoint%5C%22;0;0;0;65472%5Cn4;%5C%22SetTemperatureWarmCylinder%5C%22;100;0;0;65472%5Cn6;%5C%22SetPressureDancer%5C%22;85;0;0;65472%5Cn7;%5C%22SetEntryWiderTension%5C%22;85;0;0;65472%5Cn9;%5C%22SetExitBeltTension%5C%22;10;0;0;65472%5Cn11;%5C%22SetpointDephase%5C%22;15;0;0;65472%5Cn12;%5C%22ActSpeedDrive1%5C%22;0;0;0;65472%5Cn19;%5C%22ActSpeedDrive8%5C%22;0;0;0;65472%5Cn20;%5C%22ActTorqueDrive1%5C%22;0;0;0;65472%5Cn21;%5C%22ActTorqueDrive2%5C%22;0;0;0;65472%5Cn22;%5C%22ActTorqueDrive3%5C%22;0;0;0;65472%5Cn23;%5C%22ActTorqueDrive4%5C%22;0;0;0;65472%5Cn24;%5C%22ActTorqueDrive5%5C%22;0;0;0;65472%5Cn25;%5C%22ActTorqueDrive6%5C%22;0;0;0;65472%5Cn26;%5C%22ActTorqueDrive7%5C%22;0;0;0;65472%5Cn27;%5C%22ActTorqueDrive8%5C%22;0;0;0;65472%5Cn28;%5C%22ActSpeedMotor1%5C%22;0;0;0;65472%5Cn29;%5C%22ActSpeedMotor2%5C%22;0;0;0;65472%5Cn30;%5C%22ActSpeedMotor3%5C%22;0;0;0;65472%5Cn31;%5C%22ActSpeedMotor4%5C%22;0;0;0;65472%5Cn32;%5C%22ActSpeedMotor5%5C%22;0;0;0;65472%5Cn33;%5C%22ActSpeedMotor6%5C%22;0;0;0;65472%5Cn34;%5C%22ActSpeedMotor7%5C%22;0;0;0;65472%5Cn35;%5C%22ActSpeedMotor8%5C%22;0;0;0;65472%5Cn36;%5C%22EMU_Current%5C%22;2.25;0;0;65472%5Cn37;%5C%22EMU_Voltage%5C%22;394.4;0;0;65472%5Cn59;%5C%22ActSpeedDrive2%5C%22;0;0;0;65472%5Cn60;%5C%22ActSpeedDrive3%5C%22;0;0;0;65472%5Cn61;%5C%22ActSpeedDrive4%5C%22;0;0;0;65472%5Cn62;%5C%22ActSpeedDrive5%5C%22;0;0;0;65472%5Cn63;%5C%22ActSpeedDrive6%5C%22;0;0;0;65472%5Cn64;%5C%22ActSpeedDrive7%5C%22;0;0;0;65472%5Cn66;%5C%22ActTemperatureWarmCylinder%5C%22;36.9;0;0;65472%5Cn67;%5C%22Countermeter_total%5C%22;137266.2;0;0;65472%5Cn68;%5C%22Countermeter_daily%5C%22;282769.5;0;0;65472%5Cn69;%5C%22ActExitBeltTension%5C%22;11.64892;0;0;65472%5Cn70;%5C%22ActEntryWiderTension%5C%22;75.7561;0;0;65472%5Cn71;%5C%22ActLineSpeed%5C%22;0;0;0;65472%5Cn72;%5C%22ActPosBeltLeftPress%5C%22;-0.4934523;0;0;65472%5Cn73;%5C%22ActPosBeltRightPress%5C%22;-0.1699877;0;0;65472%22,%22url%22:%22https:%2F%2Fstatus.aws.amazon.com%2Frss%2Fall.rss%22,%22url_options%22:%7B%22method%22:%22GET%22,%22data%22:%22%22%7D,%22csv_options%22:%7B%22delimiter%22:%22;%22,%22skip_empty_lines%22:false,%22skip_lines_with_error%22:false,%22relax_column_count%22:false,%22columns%22:%22%22,%22comment%22:%22%22%7D,%22root_selector%22:%22rss.channel%5B0%5D.item%22,%22columns%22:%5B%5D,%22filters%22:%5B%5D,%22global_query_id%22:%22%22%7D%5D)?

**Comment by TFabio69** [comment]
Due to an internal error, this web page could not be loaded. We recommend that you close this page.
Microsoft Defender for Office 365 has encountered an error.
Sorry, is'nt possible to see.

Fabio

Da: Sriram ***@***.***>
Inviato: mercoledì 27 ottobre 2021 13:55
A: yesoreyeram/grafana-infinity-datasource ***@***.***>
Cc: Fabio Tiozzo ***@***.***>; Author ***@***.***>
Oggetto: Re: [yesoreyeram/grafana-infinity-datasource] Help to import HTML into Grafana (Discussion #184)


Something like this<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgrafana-infinity-datasource.herokuapp.com%2Fexplore%3ForgId%3D1%26left%3D%255B%2522now-1h%2522%2C%2522now%2522%2C%2522Infinity%2522%2C%257B%2522refId%2522%3A%2522A%2522%2C%2522type%2522%3A%2522csv%2522%2C%2522source%2522%3A%2522inline%2522%2C%2522format%2522%3A%2522table%2522%2C%2522data%2522%3A%2522%255C%2522TagId%255C%2522%3B%255C%2522TagName%255C%2522%3B%255C%2522Value%255C%2522%3B%255C%2522AlStatus%255C%2522%3B%255C%2522AlType%255C%2522%3B%255C%2522Quality%255C%2522%255Cn2%3B%255C%2522SpeedSetPoint%255C%2522%3B0%3B0%3B0%3B65472%255Cn4%3B%255C%2522SetTemperatureWarmCylinder%255C%2522%3B100%3B0%3B0%3B65472%255Cn6%3B%255C%2522SetPressureDancer%255C%2522%3B85%3B0%3B0%3B65472%255Cn7%3B%255C%2522SetEntryWiderTension%255C%2522%3B85%3B0%3B0%3B65472%255Cn9%3B%255C%2522SetExitBeltTension%255C%2522%3B10%3B0%3B0%3B65472%255Cn11%3B%255C%2522SetpointDephase%255C%2522%3B15%3B0%3B0%3B65472%255Cn12%3B%255C%2522ActSpeedDrive1%255C%2522%3B0%3B0%3B0%3B65472%255Cn19%3B%255C%2522ActSpeedDrive8%255C%2522%3B0%3B0%3B0%3B65472%255Cn20%3B%255C%2522ActTorqueDrive1%255C%2522%3B0%3B0%3B0%3B65472%255Cn21%3B%255C%2522ActTorqueDrive2%255C%2522%3B0%3B0%3B0%3B65472%255Cn22%3B%255C%2522ActTorqueDrive3%255C%2522%3B0%3B0%3B0%3B65472%255Cn23%3B%255C%2522ActTorqueDrive4%255C%2522%3B0%3B0%3B0%3B65472%255Cn24%3B%255C%2522ActTorqueDrive5%255C%2522%3B0%3B0%3B0%3B65472%255Cn25%3B%255C%2522ActTorqueDrive6%255C%2522%3B0%3B0%3B0%3B65472%255Cn26%3B%255C%2522ActTorqueDrive7%255C%2522%3B0%3B0%3B0%3B65472%255Cn27%3B%255C%2522ActTorqueDrive8%255C%2522%3B0%3B0%3B0%3B65472%255Cn28%3B%255C%2522ActSpeedMotor1%255C%2522%3B0%3B0%3B0%3B65472%255Cn29%3B%255C%2522ActSpeedMotor2%255C%2522%3B0%3B0%3B0%3B65472%255Cn30%3B%255C%2522ActSpeedMotor3%255C%2522%3B0%3B0%3B0%3B65472%255Cn31%3B%255C%2522ActSpeedMotor4%255C%2522%3B0%3B0%3B0%3B65472%255Cn32%3B%255C%2522ActSpeedMotor5%255C%2522%3B0%3B0%3B0%3B65472%255Cn33%3B%255C%2522ActSpeedMotor6%255C%2522%3B0%3B0%3B0%3B65472%255Cn34%3B%255C%2522ActSpeedMotor7%255C%2522%3B0%3B0%3B0%3B65472%255Cn35%3B%255>?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fyesoreyeram%2Fgrafana-infinity-datasource%2Fdiscussions%2F184%23discussioncomment-1544867&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7C60c9301230b14c3503f608d99940ab1b%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709325334644659%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=r88aEPnQyhTA0jLyIClzklbwKn0SmM2gqSR5091fWXM%3D&reserved=0>, or unsubscribe<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWHPPVL4SUC7KMVY4KUOB2LUI7ZDXANCNFSM5GZ5F5VA&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7C60c9301230b14c3503f608d99940ab1b%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709325334644659%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ua3chD5DqfxgJkKW0qxvHsq6O%2F7%2F7Ubu2ASQYtLjEWI%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7C60c9301230b14c3503f608d99940ab1b%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709325334654647%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Mqboj%2FUauvMZZrna7%2BqQsLSBSpPJX6lCCZ%2Fg17%2BNa6g%3D&reserved=0> or Android<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cfabio.tiozzo%40santexrimar.com%7C60c9301230b14c3503f608d99940ab1b%7Cef572d8fc35f4bf9aa033b6aec9f3678%7C0%7C0%7C637709325334654647%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=waRgbaBY4Tdcg4fsmyrZpF6J%2BfK1gJVBEkxoDLqL1y4%3D&reserved=0>.

________________________________
CONFIDENTIALITY NOTICE
This  message  and   its   attachments   are  addressed  solely  to  the  persons above  and   may  contain  confidential  information.  If  you  have  received  the message  in  error, be informed that any use of the content hereof is prohibited. Please return it immediately to the sender and delete the message. Should you have any questions, please contact us by replying to Santex Rimar ***@***.***>
Thank you

NOTIZIA CONFIDENZIALE
Questo  messaggio  e  i  suoi allegati  sono  indirizzati alle persone scritte sopra. Se  avete  ricevuto  questo  messaggio per errore, Vi informiamo che l'uso delle informazioni  contenute   è   proibito.  Prego  rinviate al  mittente immediatamente. Per altri problemi inviare una e-mail a Santex Rimar ***@***.***>
Grazie
